compiler/testData/ir/irText/expressions/smartCastsWithDestructuring.txt

```diff
@@ -49,12 +49,13 @@ FILE fqName:<root> fileName:/smartCastsWithDestructuring.kt
           then: RETURN type=kotlin.Nothing from='public final fun test (x: <root>.I1): kotlin.Unit declared in <root>'
             GET_OBJECT 'CLASS IR_EXTERNAL_DECLARATION_STUB OBJECT name:Unit modality:FINAL visibility:public superTypes:[kotlin.Any]' type=kotlin.Unit
       COMPOSITE type=kotlin.Unit origin=DESTRUCTURING_DECLARATION
-        VAR IR_TEMPORARY_VARIABLE name:tmp_0 type:<root>.I1 [val]
-          GET_VAR 'x: <root>.I1 declared in <root>.test' type=<root>.I1 origin=null
+        VAR IR_TEMPORARY_VARIABLE name:tmp_0 type:<root>.I2 [val]
```

Comment:
Please, add irText test as mentioned above (x: Any, smart cast to I1 and I2 in destructuring assignment).

Reply:
Done.